pyproject.toml

```diff
@@ -134,6 +134,7 @@ filterwarnings_when_strict = [
     "default:(Observation|Variable) names are not unique. To make them unique:UserWarning",
     "default::scipy.sparse.SparseEfficiencyWarning",
     "default::dask.array.core.PerformanceWarning",
+    "default::SyntaxWarning",  # https://github.com/linnarsson-lab/loompy/issues/183
```

Comment:
I'm against a blanket ignore on syntax warnings here. I'm okay with ignoring `loompy`'s but I don't want to ignore `SyntaxWarning`s from our codebase.

Can we just pass `--compile` to `uv pip install` instead?